documentation/Makefile

```diff
@@ -4,6 +4,12 @@ DISCOURSE_MD = $(addprefix discourse/, $(SRC))
 HTML_MD = $(addprefix html/, $(SRC))
 HTML = $(HTML_MD:.md=.html)
 
+.PHONY: new
+new: clean all
+
+.PHONY: all
+all: discourse html
+
```

Comment:
Not sure it's strictly needed, but it sounds ok to me to make `all` the new default recipe.
OTOH, I think using `new` as the default is counter-productive. We shouldn't call the clean recipe on every invocation of make IMO.

Would you mind moving these two recipes down (next to the html and discourse)?

Reply:
I think I have done as you asked, please confirm.

Reply:
I was thinking of moving the `all` recipe too but at least we don't clean by default. That will do it. Thanks

Reply:
Oh, you were suggesting the `discourse` to be the default.  I get it now, will fix.